dao/mongodb.rya/src/test/java/org/apache/rya/mongodb/MongoDBRyaDAOIT.java

```diff
@@ -41,34 +40,38 @@
 import org.junit.Before;
 import org.junit.Test;
 
+import com.mongodb.MongoClient;
 import com.mongodb.MongoException;
 import com.mongodb.client.MongoCollection;
 import com.mongodb.client.MongoDatabase;
 
-public class MongoDBRyaDAOIT extends MongoRyaTestBase {
-
+public class MongoDBRyaDAOIT {
+    private MongoClient client;
     private MongoDBRyaDAO dao;
-    private MongoDBRdfConfiguration configuration;
+    private MongoDBRdfConfiguration conf;
+
+    private static final String DB_NAME = "testInstance";
 
     @Before
     public void setUp() throws IOException, RyaDAOException{
-        final Configuration conf = new Configuration();
-        conf.set(MongoDBRdfConfiguration.MONGO_DB_NAME, getDbName());
+        client = MockMongoSingleton.getInstance();
+        client.dropDatabase(DB_NAME);
```

Comment:
I think this client.dropDatabase, and the DB_NAME could be pushed into a JUnit `@Rule` similar to what RyaTestInstanceRule does.  That way we don't forget to do it if we're using the `@Rule`.  Also cuts down on redundant code in the tests that deals with singleton specific issues.